experimental/dds/tree/src/SharedTree.ts

```diff
@@ -827,7 +827,10 @@ export class SharedTree extends SharedObject<ISharedTreeEvents> implements NodeI
 	public loadSummary(summary: SharedTreeSummaryBase): void {
 		const { version: loadedSummaryVersion } = summary;
 
-		if (isUpdateRequired(loadedSummaryVersion, this.writeFormat)) {
+		if (
```

Comment:
nit: Maybe add a comment here for this readonly check. For summarizers and for readonly clients in general, sending ops will either fail or be a no-op, both of which are not ideal.

Reply:
Right - I thought about adding a comment, but then I realized the comment I'd write would be something like "If you are a read-only client, don't send an op" and TBH that already seemed self-evident from the code.